indico/modules/events/registration/templates/emails/registration_state_update_to_managers.html

```diff
@@ -2,12 +2,46 @@
 {% block registration_body %}{% endblock %}
 
 {% block subject_message -%}
-    Registration {{ registration.state.title|lower }}
+    {% if registration.state.name == 'complete' -%}
+        {% trans title=event.title %}Registration complete for {{ title }}{% endtrans%}
```

Comment:
```suggestion
        {% trans title=event.title %}Registration complete for {{ title }}{% endtrans %}
```

same below...

Reply:
wtf i was so sure I had replaces all of them...